core/src/verification/queue/mod.rs

```diff
@@ -64,10 +64,9 @@ impl Default for Config {
 pub struct VerificationQueue<K: Kind> {
     engine: Arc<dyn CodeChainEngine>,
     verification: Arc<Verification<K>>,
-    processing: RwLock<HashMap<BlockHash, U256>>, // hash to score
+    processing: RwLock<HashMap<BlockHash, U256>>, // hash to block number
```

Comment:
It seems that we don't use the block number.
How about changing it to HashSet of the block hashes?

Reply:
Like this?
```rust
processing: RwLock<HashMap<BlockHash, U256>>, // HashSet of the block hashes
```

Reply:
like this processing: RwLock<HashSet<BlockHash>>

Reply:
I see. Thank you : )

Reply:
I modified it! : )